### DISCUSSION TITLE
How to share my project with another SUPABASE user?

### DISCUSSION THREAD

**Discussion by jfbaro** [supabase/supabase#3864] [Questions] [answered]
Hi there, I just shared my PoC project with one of my friends and he got the invitation, but the link that was sent to him was localhost:3000.

If I share my link he got a blank (black) screen.

Any help would be much appreciated.

Congrats on the amazing service guys!

**Comment by silentworks** [comment, marked as answer]
Do you mean to collaborate with you on the project through the Supabase dashboard or to actually view the project in their browser?

The localhost:3000 would be sent if your Site URL is set to that, you would need that to point to an actual domain for the invite link to be correct.

**Reply by jfbaro** [reply]
Thanks for the explanation. Yes, I would like my friends to be able to (through the browser) have access to the workspace, just like I do.  So that they can look at the tables, functions and sample data I am creating and then help me to improve my PoC.

**Reply by ttay24** [reply]
@jfbaro did you ever figure this out by chance?

Edit: oops, nevermind, figured it out!

**Reply by DonCaproni** [reply]
@ttay24 how did you figure it out?

**Reply by ttay24** [reply]
@jcaproni you need to go to your organization settings page (if you’re on the dashboard for a project, click the org name at the top of the screen to open a drop down, then click your org). The URL should be something like https://app.supabase.com/org/your-org-id/settings
Once you’re there, there’s a teams tab. Click that and then invite another user.

**Reply by DonCaproni** [reply]
Legend!

Just find it here, thanks!

**Reply by jaredpavan** [reply]
Where do I find the Teams tab? I am only seeing these sections in the settings page:

- Project Settings
-- General
-- Database
-- API
-- Auth
-- Storage
- Billing
-- Subscription
-- Usage
-- Invoices

**Reply by mayankrastogi02** [reply]
It should be under your organization settings. The URL should looks something like: 
https://supabase.com/dashboard/org/your-org-id/team

Another way to reach the Team settings is by clicking on your organization from the dropdown on the top navbar and selecting the organization name.

**Reply by ChiefJanitorialIntern** [reply]
Would there be any reason that a org invite does not make it to the end user? 

I've been invited to an org, but have yet to receive the email...

**Reply by ahsanoptiapm** [reply]
@silentworks What if I don't want the user to have global organization level access and just give access for one project, in that case what option do I have, because as mentioned before when you add users to the authentication-> Users it send an invite to that user with this content
`You have been invited
You have been invited to create a user on http://localhost:3000/. Follow this link to accept the invite:

[Accept the invite](https://xxxxxx.supabase.co/auth/v1/verify?token=xxxxxxx&type=invite&redirect_to=http://localhost:3000)`

**Reply by GaryAustin1** [reply]
@ahsanoptiapm  Only Team and Enterprise accounts have project level access.
https://supabase.com/docs/guides/platform/access-control#organization-scoped-roles-vs-project-scoped-roles
https://github.com/orgs/supabase/discussions/35172
